src/global_scheduler/global_scheduler.h

```diff
@@ -73,6 +78,9 @@ typedef struct {
   aux_address_entry *photon_plasma_map;
   /** Objects cached by this global scheduler instance. */
   scheduler_object_info *scheduler_object_info_table;
+  /** An array of tasks that are currently impossible to schedule due to their
+   *  resource requirements. */
+  UT_array *impossible_tasks;
```

Comment:
style : I think we decided that inline comments should be with a single star, since it's not meant to be part of the doxygen documentation?